Загрузка файлов с ДЗ №1.

IgushkinV opened the pull request:
Файл Main.java - Вызов методов для выполнения 15 пунктов из ДЗ №1.
Файл MyStringMethods.java - описание методов для выполнения 15 пунктов ДЗ №1.

IgushkinV committed:
Загрузка файлов с ДЗ №1.

Файл Main.java - Вызов методов для выполнения 15 пунктов из ДЗ №1.
Файл MyStringMethods.java - описание методов для выполнения 15 пунктов ДЗ №1.

tlt-java-courses commented on src/main/java/Igushkin/Lesson1/Main.java:
```diff
@@ -1 +1,38 @@
+        String[] stringsArray = methods.splitOnSpace(helloWorld); //14. Разбить строку по символу пробел
+                                                                // (т.е. чтобы каждое слово было отдельным элементом массива)
+        String reverseStr = methods.reverse("123456789"); //15. Поменять последовательность символов в строке на обратную.
+
```
Зачем нам эти пустые строки?

IgushkinV replied:
Забыл о них в конце. Убрал

tlt-java-courses commented on src/main/java/Igushkin/Lesson1/Main.java:
```diff
@@ -1 +1,38 @@
+        boolean startsWithJava = methods.isStarsWithJava("Java was created by Oracle"); //11. Проверить начинается-ли строка на "Java"
+        String replaceAO = methods.replaceAWithO("Здесь нет символа 'а'."); //12.Заменить все символы "а" в строке на символ "о"
+        String subStr = methods.substringFrom44to90(longString); //13. Получить подстроку с 44 символа по 90 символ
+        //System.out.println(subStr);
```
Для чего нам закомментированная строка?

IgushkinV replied:
Убрал. Была для вывода подстроки с 44 по 90 символ. В задании не было именно "вывести" эту подстроку, было "получить".

tlt-java-courses commented on src/main/java/Igushkin/Lesson1/MyStringMethods.java:
```diff
@@ -0,0 +1,141 @@
+
+public class MyStringMethods {
+
+    /**
```
Давай сделаем комментарии в формате JavaDoc?

IgushkinV replied:
Переделал. Добавил описание класса, теги @param и @return к каждому методу.

tlt-java-courses commented on src/main/java/Igushkin/Lesson1/MyStringMethods.java:
```diff
@@ -0,0 +1,141 @@
+    /**
+     * Создаёт новую строку с помощью конструктора и заносит ее в пул литералов
+     */
+
```
Для чего эти пустые строки?

IgushkinV replied:
Убрал.

IgushkinV committed:
Update Main.java

Удаление пустых и закомментированных строк.

IgushkinV committed:
Редактирование комментариев в стиле javaDoc в MyStringMethods.java

Изменил комментарии, добавил теги @param и  @return. 
Убрал все лишние пустые строки.
